tests/integrations/test_dbm.py

```diff
@@ -5,19 +5,32 @@
 from utils import weblog, interfaces, context, missing_feature, released, scenario
 
 
-@released(cpp="?", golang="?", java="?", dotnet="?", nodejs="?", php="?", ruby="?")
-@missing_feature(context.library == "python" and context.weblog_variant != "flask-poc", reason="Missing on weblog")
+@released(cpp="?", golang="?", java="?", nodejs="?", dotnet="2.24.0", php="?", ruby="?")
+@missing_feature(
+    context.library in ["python", "nodejs"] and context.weblog_variant != "flask-poc", reason="Missing on weblog",
+)
 @scenario("INTEGRATIONS")
 class Test_Dbm:
     """Verify behavior of DBM propagation"""
 
     def setup_trace_payload(self):
-        self.requests = [
-            weblog.get("/dbm", params={"integration": "psycopg", "cursor_method": "execute"}),
-            weblog.get("/dbm", params={"integration": "psycopg", "cursor_method": "executemany"}),
-        ]
+        self.library_name = context.library
+        self.requests = []
+
+        if self.library_name == "python":
+            self.requests = [
+                weblog.get("/dbm", params={"integration": "psycopg", "operation": "execute"}),
+                weblog.get("/dbm", params={"integration": "psycopg", "operation": "executemany"}),
+            ]
+        elif self.library_name == "dotnet":
+            self.requests = [
+                weblog.get("/dbm", params={"integration": "mysql"}),
+                weblog.get("/dbm", params={"integration": "npgsql"}),
+            ]
 
     def test_trace_payload(self):
+        assert self.requests, "Requests were not submit for {}. Please disable this test for this library "
```

Comment:
`assert <array>` is always true (and actually, requests in this array will always be submitted). 

I think the test above is enough.

Reply:
Thank you for the review, I removed it then ran the tested that the tests still passed:
https://github.com/DataDog/system-tests/pull/908/commits/59f875455b48a572074988cb348a63dd0da44632